test/purchases.test.js

```diff
@@ -83,6 +82,7 @@ describe('POST /albums/:id', () => {
         )
         .then(response => {
           validToken = response.text;
+          done();
```

Comment:
This `done` was from when we were trying to make this work right? If so, then it is deprecated and should be removed.